crates/typed-store/src/rocks/mod.rs

```diff
@@ -1825,14 +1864,40 @@ pub fn base_db_options() -> DBOptions {
     }
 
     // The table cache is locked for updates and this determines the number
-    // of shards, ie 2^6. Increase in case of lock contentions.
+    // of shards, ie 2^10. Increase in case of lock contentions.
     opt.set_table_cache_num_shard_bits(10);
 
+    // Allow buffering 1GiB of writes before flushing to disk.
+    opt.set_write_buffer_size(256 * 1024 * 1024);
+    opt.set_max_write_buffer_number(4);
+
+    opt.set_universal_compaction_options(&UniversalCompactOptions::default());
+    // Compaction trigger for universal compaction. Default to 8.
```

Comment:
Are we moving to universal compaction by default? This is a pretty impacting change. Are we sure about this?

Reply:
I have not had time to test universal vs none universal, with all parameters being the same. But my intuition is that using universal compaction will have slight benefits / neutral, with large write buffers and level 0. So far I have not observed any issue in workloads.

Reply:
Overall this configuration does look better for batched workload, without pruning. I'm not sure if there is a good equivalent setting for non-universal. With universal policy, IIUC more data will be accumulated in L0 compared to non-universal. After that things should work the same.

I think the biggest problem right now is that for objects / effects and other perpetual tables, our changes to write buffer limits do not seem to raise their total write buffer sizes that much. Once we fix that, on disk compaction policies will become more important.

Reply:
ok so the compaction policy is still `level` looks like because we are not setting `opt.set_compaciton_style` to universal, so this is just option is not really needed `opt.set_universal_compaction_options(&UniversalCompactOptions::default());`?

Reply:
Good point. I think the most recent experiments probably are not testing universal policy. Removing.